### DISCUSSION TITLE
Trigger ajax cart update from javascript.

### DISCUSSION THREAD

**Discussion by manuelhpineda** [EvgeniyMukhamedjanov/liquid-ajax-cart#100] [Q&A] [answered]
Hello, is there a feature to trigger an update from outside the ajax cart. For example I want to open a cart drawer and trigger and update event for the ajax cart to do the normal rendering as if I am adding a product from ajax cart.

**Comment by manuelhpineda** [comment]
so what I am doing not sure if this is the best option but I am triggering this event to update the cart:

`            document.dispatchEvent(
              new CustomEvent("liquid-ajax-cart:init", {
                detail: {
                  // Add any data you want to pass with the event
                  cart: window.liquidAjaxCart.cart,
                },
              }),
            )
`

**Reply by EvgeniyMukhamedjanov** [reply, marked as answer]
Hello @manuelhpineda 
If I understood you right, you just need to call window.liquidAjaxCart.update() method.

**Reply by manuelhpineda** [reply]
thanks a lot that basically solved it
